x-pack/filebeat/module/gcp/cloud_armor/config/pipeline.js

```diff
@@ -0,0 +1,244 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+function Audit(keep_original_message) {
+    var processor = require("processor");
+
+    // The pub/sub input writes the Stackdriver LogEntry object into the message
+    // field. The message needs decoded as JSON.
+    var decodeJson = new processor.DecodeJSONFields({
+        fields: ["message"],
+        target: "json",
+    });
+
+    // Set @timetamp the LogEntry's timestamp.
+    var parseTimestamp = new processor.Timestamp({
+        field: "json.timestamp",
+        timezone: "UTC",
+        layouts: ["2006-01-02T15:04:05.999999999Z07:00"],
+        tests: ["2019-06-14T03:50:10.845445834Z"],
+        ignore_missing: true,
```

Comment:
Do we want to ignore missing in this case given it's a timestamp? Or is this more for testing at the moment?

Reply:
I think I followed what the GCP audit fileset was doing out of the box in the `gcp` module but you're probably right!